### PR TITLE
Candidate_List: Display Scanner and Human entity types, and add to filter: Redmine 11022

### PR DESCRIPTION
**Pull request by MounaSafiHarab**

### CODE REVIEW

**Comment by MounaSafiHarab** [comment]
@driusan 
this needs to be rebased on @stellarxo latest changes on candidate list...
but I am sending it here to make sure this is how it is to be integrated.

File: modules/candidate_list/php/NDB_Menu_Filter_candidate_list.class.inc [hunk 1]
```diff
@@ -126,7 +127,9 @@ class NDB_Menu_Filter_Candidate_List extends NDB_Menu_Filter
                 ON (ps.participant_status=pso.ID)
             LEFT JOIN Project p ON (c.ProjectID=p.ProjectID)
             LEFT JOIN subproject sp ON (s.SubprojectID=sp.SubprojectID)
-            WHERE c.Entity_type = 'Human' AND c.Active = 'Y'";
+            WHERE (c.Entity_type = 'Scanner' OR c.Entity_type = 'Human')
```
Comment (Jkat) [inline modules/candidate_list/php/NDB_Menu_Filter_candidate_list.class.inc]: Alternatively, you can just get rid of c.Entity_type

Comment (driusan) [inline modules/candidate_list/php/NDB_Menu_Filter_candidate_list.class.inc]: I think @Jkat is right. The field is defined as `enum('Human', 'Scanner') NOT NULL` in the schema.. so this clause is meaningless.

Comment (Jkat) [inline modules/candidate_list/php/NDB_Menu_Filter_candidate_list.class.inc]: I think you have a syntax error because it says "WHERE AND c.Active = 'Y'"

**Comment by christinerogers** [comment]
Is there a way to make this optional/configurable/disable-able for projects that don't have imaging data?

**Comment by MounaSafiHarab** [comment]
@christinerogers 

By default, all candidates are inserted into the candidate table as Human entities, irrespective of whether there is imaging data or not (making the filter still a valid one; it is just that you would get no entries if you filter for Scanner and get all candidate Entries if you filter for Human).

**Pull request merged**